usaspending_api/search/v2/views/spending_by_category_views/spending_by_federal_account.py

```diff
@@ -0,0 +1,67 @@
+import json
+from abc import ABCMeta
+from decimal import Decimal
+from enum import Enum
+from typing import List
+
+from django.db.models import QuerySet
+
+from usaspending_api.common.exceptions import InvalidParameterException
+from usaspending_api.search.v2.views.spending_by_category_views.spending_by_category import (
+    Category,
+    AbstractSpendingByCategoryViewSet,
+)
+
+
+class AccountType(Enum):
+    FEDERAL_ACCOUNT = "federal_account"
+
+
+class AbstractAccountViewSet(AbstractSpendingByCategoryViewSet, metaclass=ABCMeta):
+    """
+    Abstract class used by the different Awarding / Funding Agencies and Subagencies
+    """
+
```

Comment:
Need to add the `endpoint_doc` property to the class that points to the API contract.

Reply:
It's in the `FederalAccountViewSet` class

Reply:
Woops I missed that.